app/services/noko_service.rb

```diff
@@ -23,29 +23,38 @@ def self.import_projects
 
   def self.import_entries(start_date, end_date)
     result = client.get_entries(from: start_date, to: end_date)
-    save_entries_for(result)
+    missing_projects = []
+    save_entries_for(result, missing_projects)
 
     if last = result.try(:link).try(:last)
       last_page = last.match(/page=(\d+)/)[1].to_i
       (2..last_page).each do |page|
         result = client.get_entries(from: start_date, to: end_date, page: page)
-        save_entries_for(result)
+        save_entries_for(result, missing_projects)
```

Comment:
Maybe we can do `client.get_entries(from: start_date, to: end_date, per_page: 1000)` so we don't have to do all this. I think `1000` entries is the maximum we can fetch per page, but i don't see that's a problem for us. What you think?